### ISSUE TITLE
Project(swift 5) doesn't build after installation RxFlow

### ISSUE THREAD

**Issue by bobkexit** [RxSwiftCommunity/RxFlow#119]
PodFile:

```
target 'TimeTimer' do
  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for TimeTimer
  pod 'RealmSwift'
  pod 'Fabric'
  pod 'Crashlytics'
  pod 'RxSwift', '~> 5'
  pod 'RxCocoa', '~> 5'
  pod 'RxFlow'

  target 'TimeTimerTests' do
    inherit! :search_paths
    # Pods for testing
    pod 'RxBlocking', '~> 5'
    pod 'RxTest', '~> 5'
  end

end
```

After installation RxFlow, I tried build project and got error:

![image](https://user-images.githubusercontent.com/5675375/57151044-fe6a6c00-6de0-11e9-933d-eb59c5ce4625.png)

**Comment by Willicious-k** [comment]
Same error happened to me in both versions (2.2.0 / 2.1.0)

**Comment by Willicious-k** [comment]
https://github.com/RxSwiftCommunity/RxFlow/pull/118/files#diff-6236ced41531433350267fc331031515R67

Seems like this issue has been already fixed, but not released yet.

**Comment by twittemb** [comment]
Hi There

I will release a version with the merged PR asap.

Stay tuned

**Comment by twittemb** [comment]
Can you try in the last release please ?

**Comment by Willicious-k** [comment]
Build error has resolved in 2.3.0. (with RxSwift ~> 5.0.0)

Thank you @twittemb

**Comment by twittemb** [comment]
Nice to hear 👌
